docs/admin/cluster-large.md

```diff
@@ -40,35 +40,63 @@ To avoid running into cloud provider quota issues, when creating a cluster with
     * Target pools
 * Gating the setup script so that it brings up new node VMs in smaller batches with waits in between, because some cloud providers rate limit the creation of VMs.
 
-### Etcd storage
-
-To improve performance of large clusters, we store events in a separate dedicated etcd instance.
-
-When creating a cluster, existing salt scripts:
-
-* start and configure additional etcd instance
-* configure api-server to use it for storing events
-
+### Etcd storage
+
+To improve performance of large clusters, we store events in a separate dedicated etcd instance.
+
+When creating a cluster, existing salt scripts:
+
+* start and configure additional etcd instance
+* configure api-server to use it for storing events
+
+### Size of master and master components
+
+On GCE/GKE and AWS, `kube-up` automatically configures the proper VM size for your master depending on the number of nodes
+in your cluster. On other providers, you will need to configure it manually. For reference, the sizes we use on GCE are
+* 1-5 nodes: n1-standard-1
+* 6-10 nodes: n1-standard-2
+* 11-100 nodes: n1-standard-4
+* 101-250 nodes: n1-standard-8
+* 251-500 nodes: n1-standard-16
+* more than 500 nodes: n1-standard-32
+
+And the sizes we use on AWS are
+* 1-149 nodes: m3.medium
+* more than 150 nodes: m3.large
+

```

Comment:
Actually I think the SKUs is more useful because that tells people how much it's going to cost. And the other table (with the 99th percentile resource usages) is useful for knowing how much resources to provision if you're on bare metal.